src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -83,20 +83,20 @@ msgstr "Kalibriere XY"
 #. Back
 #: src/gui/screen_sheet_rename.cpp:18 src/gui/dialogs/dialog_response.cpp:13
 msgid "CANCEL"
-msgstr "Abbruch"
+msgstr "ABBRUCH"
 
 #: src/gui/dialogs/DialogFactory.cpp:12 src/gui/screen_menu_filament.cpp:90
 msgid "CHANGE FILAMENT"
-msgstr "Filamentwechsel"
+msgstr "FILAMENTWECHSEL"
 
 #: src/gui/screen_menu_filament.cpp:89 src/guiapi/include/MItem_tools.hpp:279
 msgid "Change Filament"
-msgstr "Filament wechseln"
+msgstr "Filamentwechsel"
```

Comment:
I think the origin translation is better because the verb shows that it is an activity (though of nativ speaker)